ncep/packages/gptl/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
I am trying to move the package file for GPTL to the builtin spack package list.

See https://github.com/spack/spack/pull/25993

Reply:
This has been accepted into spack, so I will remove it from here.